web/src/routes/TuNewsDetailPage.tsx

```diff
@@ -69,7 +69,12 @@ const TuNewsDetailPage = ({ city, pathname, cityCode, languageCode }: CityRouteP
   if (loading) {
     return (
       <CityContentLayout isLoading {...locationLayoutParams}>
-        <LoadingSpinner />
+        <TuNewsBanner>
+          <IconContainer width={180} height='100%'>
+            <Svg src={TuNewsActiveIcon} width='100%' height='100%' />
+          </IconContainer>
+        </TuNewsBanner>
+        <SkeletonPage />
       </CityContentLayout>
     )
   }
```

Comment:
🤔 Same as for `LocalNewsPage`, what do you think of removing this `if (loading)` block and instead moving `SkeletonPage` to L111 and `SkeletonList` down to 108 in the same way we did for events and landing in #3379?
This allows us to remove some conditionals and avoids duplicating the `TuNewsBanner`.